TombEngine/Objects/TR4/Object/WraithTrap.cpp

```diff
@@ -22,8 +22,8 @@ namespace TEN::Entities::TR4
 	{
 		auto& item = g_Level.Items[itemNumber];
 
-		item.Animation.AnimNumber = Objects[item.ObjectNumber].animIndex;
-		item.Animation.FrameNumber = g_Level.Anims[item.Animation.AnimNumber].frameBase;
+		item.Animation.AnimNumber = Objects[item.ObjectNumber].animIndex + 3;
```

Comment:
Why a magic number 3 was added here?

Reply:
May have been a stray Crtl+V. Should use `SetAnimation()` here.

Reply:
Yes please :)